packages/baseui-codemods/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uber-web-ui/baseui-codemods",
-  "version": "1.0.9",
+  "version": "2.0.2",
```

Comment:
Why `2.0.2`?

Reply:
i published 2.0.0 and 2.0.1 in the original pr - so i had to skip a couple numbers here.